keyboards/mehkee96/config.h

```diff
@@ -12,12 +13,6 @@
 #define MATRIX_COLS 15
 
 #define RGBLED_NUM 16
-#define RGBLIGHT_ANIMATIONS
```

Comment:
removing this line causes a build failure:
 | .build/obj_mehkee96_johann/keyboards/mehkee96/mehkee96.o: In function `matrix_scan_kb':
 | /home/johann/qmk_firmware/keyboards/mehkee96/mehkee96.c:61: undefined reference to `rgblight_task'
 | collect2: error: ld returned 1 exit status